metadata-fixer/src/main/java/org/verapdf/metadata/fixer/impl/pb/schemas/DublinCoreSchemaImpl.java

```diff
@@ -0,0 +1,75 @@
+package org.verapdf.metadata.fixer.impl.pb.schemas;
+
+import org.apache.xmpbox.XmpConstants;
+import org.apache.xmpbox.schema.DublinCoreSchema;
+import org.apache.xmpbox.type.AbstractField;
+import org.apache.xmpbox.type.ArrayProperty;
+import org.apache.xmpbox.type.Cardinality;
+import org.apache.xmpbox.type.TextType;
+import org.verapdf.metadata.fixer.entity.Metadata;
+import org.verapdf.metadata.fixer.schemas.DublinCore;
+
+import java.util.List;
+
+/**
+ * @author Evgeniy Muravitskiy
+ */
+public class DublinCoreSchemaImpl implements DublinCore {
+
+	private static final String AUTHOR = "creator";
+
+	private final DublinCoreSchema schema;
+	private final Metadata metadata;
+
+	public DublinCoreSchemaImpl(DublinCoreSchema schema, Metadata metadata) {
+		this.schema = schema;
+		this.metadata = metadata;
+	}
+
+	@Override
+	public String getTitle() {
+		return this.schema.getTitle();
+	}
+
+	@Override
+	public void setTitle(String title) {
+		this.schema.setTitle(title);
+	}
+
+	@Override
+	public String getSubject() {
+		return this.schema.getDescription();
+	}
+
+	@Override
+	public void setSubject(String description) {
+		this.schema.setDescription(description);
+	}
+
+	@Override
+	public String getAuthor() {
+		List<String> creators = this.schema.getCreators();
+		return creators != null && creators.size() > 0 ? creators.get(0) : null;
+	}
+
+	@Override
+	public void setAuthor(String creator) {
+		ArrayProperty seq = (ArrayProperty) schema.getAbstractProperty(AUTHOR);
+		ArrayProperty newSeq = schema.createArrayProperty(AUTHOR, Cardinality.Seq);
+		TextType li = schema.createTextType(XmpConstants.LIST_NAME, creator);
+		newSeq.getContainer().addProperty(li);
```

Comment:
Container always not null:
public AbstractComplexProperty(XMPMetadata metadata, String propertyName)
    {
        super(metadata, propertyName);
        container = new ComplexPropertyContainer();
        this.namespaceToPrefix = new HashMap<String, String>();
    }